mapper/mapper_test.go

```diff
@@ -0,0 +1,88 @@
+package mapper
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+)
+
+func TestIsClassifiedByMappedCorrectly(t *testing.T) {
+
+	originalBody, err := ioutil.ReadFile("test_resources/annotations_isClassifiedBy_v2.json")
+	if err != nil {
+		panic(err)
+	}
+	expectedBody, err := ioutil.ReadFile("test_resources/annotations_isClassifiedBy_PAC.json")
+
+	if err != nil {
+		panic(err)
+	}
+	actualBody, _ := ConvertPredicates(originalBody)
+
+	log.Info(string(actualBody))
+	log.Info(string(expectedBody))
+
+	assert.JSONEq(t, string(expectedBody), string(actualBody), "they do not match")
+}
+
+func TestIsPrimariTestConnectionErrorlyClassifiedByMappedCorrectly(t *testing.T) {
+
+	originalBody, err := ioutil.ReadFile("test_resources/annotations_isPrimarilyClassifiedBy_v2.json")
+	if err != nil {
+		panic(err)
+	}
+	expectedBody, err := ioutil.ReadFile("test_resources/annotations_isPrimarilyClassifiedBy_PAC.json")
+
```

Comment:
Are we keeping isPrimarilyClassified as an annotation for types that are not Topics or Locations ?